sdks/python/setup.py

```diff
@@ -30,8 +30,7 @@
 
 def get_version():
   global_names = {}
-  execfile(os.path.normpath('./apache_beam/version.py'),
-           global_names)
+  exec(open(os.path.normpath('./apache_beam/version.py')).read(), global_names)
```

Comment:
It should work as a relative import; many packages do that. E.g. https://github.com/cython/cython/blob/master/setup.py#L254